src/chart/graph/GraphSeries.ts

```diff
@@ -229,6 +232,28 @@ export interface GraphSeriesOption
      * auto curveness for multiple edge, invalid when `lineStyle.curveness` is set
      */
     autoCurveness?: boolean | number | number[]
+
+    thumbnail?: {
+        show?: boolean,
+
+        top?: number | string,
+
+        bottom?: number | string,
+
+        left?: number | string,
+
+        right?: number | string,
+
+        scale?: number,
+
+        borderColor?: Color,
```

Comment:
We usually use an option `itemStyle` to configure all the styles. Only `borderColor` and `backgroundColor` is not enough.